src/util/octokit.ts

```diff
@@ -18,9 +18,19 @@ export const paginate = (
   while (octokit.hasNextPage(response)) {
     // Avoid GitHub's "abuse detection mechanisms"
     await wait(2000);
```

Comment:
Should this wait be left at 2s or should this start being more aggressive now that the rate limiting catch is in place?